README.md

```diff
@@ -69,44 +69,13 @@ So, new notebooks will be saved there, unless you change the directory in the fi
 To change the default directory, you will need to specify `ServerApp.root_dir` by adding this line to previous command: `start-notebook.py --ServerApp.root_dir=/home/jovyan/work`.
 ```
 
-## Contributing
-
-Please see the [Contributor Guide on ReadTheDocs](https://jupyter-docker-stacks.readthedocs.io/en/latest/)
-for information about how to contribute recipes, features, tests, and community maintained stacks.
-
-## Maintainer Help Wanted
```

Comment:
This section doesn't make a lot of difference.

I want to move user-related sections of the readme up, so people first see how to _use_ our images, and then how to _contribute_ and what alternatives exist.